composer.json

```diff
@@ -17,10 +17,11 @@
     }
   ],
   "require": {
-    "php": ">=8.0",
+    "php": ">=8.1",
```

Comment:
This is probably not necessary if we do not require PHPUnit 10 as I suggest after.

Reply:
done: https://github.com/arderyp/panther/commit/5cf344a56fbadfced1d5f538b20a805803117b3f